drivers/artnet.js

```diff
@@ -13,58 +13,58 @@ function ArtnetDriver(deviceId = '127.0.0.1', options = {}) {
   self.universe.fill(0);
 
   /**
-   * Allow artnet rate to be set and default to 44Hz 
+   * Allow artnet rate to be set and default to 44Hz
    * @type Number
    */
   self.sleepTime = !isNaN(options.dmx_speed) ? 1000 / options.dmx_speed : 24;
 
   self.universe_id.writeInt16LE(options.universe || 0, 0);
   self.host = deviceId;
   self.port = options.port || 6454;
-  self.dev = dgram.createSocket('udp4');
+  self.dev = dgram.createSocket("udp4");
   self.dev.bind(() => self.dev.setBroadcast(true));
   self.start();
 }
 
-ArtnetDriver.prototype.sendUniverse = function () {
+ArtnetDriver.prototype.sendUniverse = function() {
   const pkg = Buffer.concat([
     this.header,
     this.sequence,
     this.physical,
     this.universe_id,
     this.length,
-    this.universe.slice(1),
+    this.universe.slice(1)
   ]);
 
   this.dev.send(pkg, 0, pkg.length, this.port, this.host);
 };
 
-ArtnetDriver.prototype.start = function () {
+ArtnetDriver.prototype.start = function() {
   this.timeout = setInterval(this.sendUniverse.bind(this), this.sleepTime);
 };
 
-ArtnetDriver.prototype.stop = function () {
+ArtnetDriver.prototype.stop = function() {
```

Comment:
I don't see the benefit of these changes.
If we change this file, we should convert the code to use modern arrow syntax and not just reformatting the old ES5 syntax.
e.g. `ArtnetDriver.prototype.stop = () => clearInterval(this.timeout);`